web-console/3-project-details/add-to-project.md

```diff
@@ -11,6 +11,16 @@
 - Selecting an item from the search will initiate the ordering flow on the [Overlay Panel](http://openshift.github.io/openshift-origin-design/web-console/4-patterns/overlay-panel) on the projects detail page.
 - The Project selection step should be hidden from the ordering panel in this case since the user is adding from within the console and we know what Project.
 
+### Browse Catalog
+![project browse](img/project-browse-01.png)
+- Selecting `Browse Catalog` from the dropdown or clicking the link to view more search results will bring up a version of the catalog embedded in the page.
+- Clicking the X button, or the "return to project" link will remove the catalog view and take the user back to where they were when they opened the catalog.
+
+![project browse](img/project-browse-02.png)
+- A filtered view of the catalog will be brought up if the quick add bar is used to trigger the catalog.
+
```

Comment:
Is the navigation bar still active? Will navigating end browsing the catalog?

Reply:
Based on the conversation that we had at the review last week, I would say that the navigation should still be functional and navigating to a different page would close the catalog. I can update this to reflect that